golden_response/staging/get_triples/stats_var.json

```diff
@@ -32,14 +32,6 @@
       "objectTypes": ["Class"],
       "provenanceId": "dc/5l5zxr1"
     },
-    {
-      "subjectId": "MarriedPopulation",
```

Comment:
can you update the stats var in get_triples_test.go to use the new stats var name: "Count_Person", "Count_Person_Female"

Reply:
Done.